client/web/web.go

```diff
@@ -283,6 +284,11 @@ func (s *Server) serve(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	if strings.HasPrefix(r.URL.Path, "/metrics") {
+		varz.Handler(w, r)
```

Comment:
This will expose everything under expvar, we might only want to expose the new metrics in this, thoughts?